go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/grafana
 
 go 1.13
 
+replace github.com/grafana/grafana-plugin-sdk-go => github.com/grafana/grafana-plugin-sdk-go v0.34.1-0.20200407094211-e32a9ffaf357
```

Comment:
Why the need for a replace directive?

Reply:
It's just temporary

Reply:
Targeting branch in sdk

Reply:
You get the same effect by upgrading the dependency version and dropping the replace directive (I just tested) :)

Reply:
I know, but this is the suggested workflow with go modules as I understand it. https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive

Reply:
Ah, I think I understand the use case now; to pin the dependency version, since Go can upgrade it if it detects a newer version? Go modules are way too magical :man_facepalming:

Reply:
Yeah exactly :smile: